content/posts/2016-03-22-what-was-your-path-to-18f.md

```diff
@@ -143,7 +143,7 @@ _The acquisitions team provides innovative services and solutions to support dig
   “Lots of time in the non-profit world with a focus on setting up systems for people to do their jobs better — not realizing until fairly recently there were entire professions of the skills I was using (user research, user experience, information architecture, curriculum development). After meeting 18F-ers I realized there were big teams of people who loved helping other people do their work better. <3”  - {{ "jessie-posilkin" | team_link }}
   </blockquote>
 </div>
-<div class="pquote">{{ "rebecca-refoy-sidibe" | team_photo }}
+<div class="pquote">{{ "rebecca-refoy" | team_photo }}
   <blockquote>“My path to 18F started with the Peace Corps. I served in Guinea, West Africa and was evacuated in the midst of the Ebola crisis. Returned Peace Corps Volunteers have one year of Non Competitive Eligibility, which makes the hiring process in the federal government much easier. I met my now-supervisor at a career fair that the Peace Corps Headquarters holds twice a year. Five months after being evacuated, I was starting in the Office of Citizen Services and Innovative Technologies. I had never heard of OCSIT or 18F before that. I have a few family members working in various parts of the government who have always regaled me with stories of the maddening bureaucracy. It was encouraging to see an entire office dedicated to hacking the government and helping them to work better. I have been working 50 percent of my time with 18F since November 2015 and have found many more qualities (transparency, professional development, empowerment, focus on diversity, etc) that are inspiring and are refreshing for the public sector.” – Rebecca Refoy-Sidibe</blockquote>
```

Comment:
clarification: If we're changing the slug to have the correct name, should we also change the attribution at the end of the quote here?

```diff
- the public sector.” – Rebecca Refoy-Sidibe</
+ the public sector.” – Rebecca Refoy</
```

Reply:
I can make that change shortly.